### PR TITLE
[Rustc Book] Explain --cfg's arguments

### PR DESCRIPTION
**Pull request by Havvy**
I removed this from the reference since it's rustc specific, and noticed it wasn't well documented on the page that should document it well. It does seem to go against the grain of one line per command line flag though.

r? @GuillaumeGomez

### CODE REVIEW

**Comment by steveklabnik** [comment]
“The grain” here isn’t intentional, it’s an artifact of me just trying to get the basics in place, and not having done much else yet.

File: src/doc/rustc/src/command-line-arguments.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ This flag will print out help information for `rustc`.
 
 This flag can turn on or off various `#[cfg]` settings.
 
+The value can either be a single identifier or two identifiers separated by `=`.
+
+For examples, `--cfg 'verbose'` or `--cfg 'feature=serde'`. These correspond
```
Comment (petrochenkov) [inline src/doc/rustc/src/command-line-arguments.md]: `--cfg feature=serde` is incorrect, you need a string literal for the value - `--cfg feature = "serde"`.

Comment (petrochenkov) [reply src/doc/rustc/src/command-line-arguments.md]: `--cfg` argument (after shell preprocessing, etc) is literally a meta-item, same thing as in `#[cfg(feature = "serde")]` and with the same restrictions.

Comment (Havvy) [reply src/doc/rustc/src/command-line-arguments.md]: Whoops. Thought I put those quotes in there. Should have just copy-pasted from the `cargo build --verbose` output...

**Comment by GuillaumeGomez** [comment]
Seems good to me. r=me once CI is green. Thanks!

**Pull request merged**